src/ngx_rewrite_driver_factory.cc

```diff
@@ -151,6 +151,10 @@ RewriteOptions* NgxRewriteDriverFactory::NewRewriteOptions() {
   return options;
 }
 
+RewriteOptions* NgxRewriteDriverFactory::NewRewriteOptionsForQuery() {
+  return new NgxRewriteOptions(thread_system());
```

Comment:
create_dir_config is -- I think -- the only place in Apache where we establish a default of kCoreFilters.  So I was thinking that we should find the equivalent place in nginx and make that, also, the only place where we establish the default of kCoreFilters.